app-text/asciidoc/asciidoc-9999.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2016 Gentoo Foundation
```

Comment:
Strictly speaking, you should have bumped the copyright on the first commit touching that file. But feel free not to do that if you don't like fighting rebases a lot ;-).